### PR TITLE
Update v3.26 release notes

### PR DESCRIPTION
**Pull request by vikaspotluri123**

### CODE REVIEW

File: src/release-notes.md [hunk 1]
```diff
@@ -8,6 +8,28 @@ description: See what's changed as Gradebook evolves!
...
+ - Loading the import page will not crash
+
+ - Trying to import a course after logging in will not cause a crash
+
+2. Logging in to Gradebook as a student in a different school will work
+
+ - Previously, the app would try to log you in, fail, and ask you to log in again
+
+3. Account creation after trying to import a course will work now
+ - Previously, you could get a cryptic error
```
Comment (joshcosta) [inline src/release-notes.md:30]: /n

File: src/release-notes.md [hunk 2]
```diff
@@ -16,35 +38,33 @@ _Released August 14, 2022_
 
 - Upcoming semesters can now be created much earlier
 
-- The semester used for GPA will always be the oldest of current semesters. This means during times when the semester is changing, such as from Summer to Fall, the Summer semester will still be used for GPA until it is archived.
+- The semester used for GPA will always be the oldest of current semesters.
 
-- Enhanced the precision of most numerical calculations
+  - During transition periods, such as between the Fall and Winter semester, the Fall semester will be used for GPA calculations until it is archived.
```
Comment (joshcosta) [inline src/release-notes.md:43]:
```suggestion
  - During transition periods, such as between the Fall and Winter semester, the older semester will be used for GPA calculations until it is archived.
```

File: src/release-notes.md (hunk 1 above)
Comment (joshcosta) [inline src/release-notes.md:30]:
```suggestion
3. Account creation after trying to import a course will work now

 - Previously, you could get a cryptic error
```

File: src/release-notes.md [hunk 3]
```diff
@@ -8,6 +8,28 @@ description: See what's changed as Gradebook evolves!
...
+
+## Bug Fixes
+
+1. Importing a course should now be fully functional
+
+ - Loading the import page will not crash
+
+ - Trying to import a course after logging in will not cause a crash
+
+2. Logging in to Gradebook as a student in a different school will work
```
Comment (joshcosta) [inline src/release-notes.md:25]:
```suggestion
2. Logging in to a school's site on Gradebook will properly take you to the correct school
```

**Review by joshcosta — APPROVED** [review]
Approved after requested changes

File: src/release-notes.md (hunk 3 above)
Comment (vikaspotluri123) [reply src/release-notes.md:25]: a `different` school on Gradebook will take you to `your` correct school?

Comment (vikaspotluri123) [reply src/release-notes.md:25]: Whatever you decide, please do a quick UI review before merging!

Comment (joshcosta) [reply src/release-notes.md:25]: I like your wording

**Pull request merged**